app/invocation/invocation_action_input_node.tsx

```diff
@@ -5,16 +5,16 @@ import DigestComponent from "../components/digest/digest";
 import format from "../format/format";
 
 interface Props {
-  node: InputNode;
+  node: Node;
   treeShaToExpanded: Map<string, boolean>;
-  treeShaToChildrenMap: Map<string, InputNode[]>;
+  treeShaToChildrenMap: Map<string, Node[]>;
   treeShaToTotalSizeMap: Map<string, [Number, Number]>;
   handleFileClicked: any;
 }
 
 interface State {}
 
-export interface InputNode {
+export interface Node {
```

Comment:
nit: "Node" feels slightly generic, maybe call it "TreeNode".

Reply:
Great suggestion.